README.md

```diff
@@ -21,6 +21,34 @@ There are two possible paths, depending on your choice of Kubernetes:
 1. For any Kubernetes, follow the [installation instructions][install].
 1. On MicroK8s, you simply have to enable the [Kubeflow add-on][addon].
 
+## Testing
+
+To deploy this bundle and run tests locally, do the following:
+1. Set up Kubernetes, Juju, and deploy the bundle you're interested in (`kubeflow` or `kubeflow-lite`) using the [installation guide](https://charmed-kubeflow.io/docs/install/)
+2. Install test prerequisites:
+```bash
+   sudo snap install juju-wait --classic
```

Comment:
These instructions are still missing  juju-kubectl.

Reply:
I'm so confused about how this ran before.  Apparently copying and pasting from integration installs isn't enough haha